alerts/package.json

```diff
@@ -4,10 +4,8 @@
   "main": "alerts.js",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.26.1"
-  },
-  "devDependencies": {
-    "aws-sdk": "^2.1099.0"
+    "@aws-sdk/client-ssm": "3.354.0",
```

Comment:
should these now live in `dependencies` to match `/slack`?

Reply:
Ah yes they do already, it's a bit noisy to see in this diff

Reply:
ah duh yes of course :)